DependencyInjection/FOSUserExtension.php

```diff
@@ -67,10 +67,14 @@ public function load(array $configs, ContainerBuilder $container)
             $definition->setFactory(array(new Reference('fos_user.doctrine_registry'), 'getManager'));
         }
 
-        foreach (array('validator', 'security', 'util', 'mailer', 'listeners') as $basename) {
+        foreach (array('validator', 'security', 'util', 'mailer') as $basename) {
             $loader->load(sprintf('%s.xml', $basename));
         }
 
+        if ($config['use_authentication_listener']) {
+            $loader->load('listeners.xml');
```

Comment:
this is indeed wrong. You should disable only the relevant listener (by removing the service)